promql/testdata/functions.test

```diff
@@ -49,3 +49,16 @@ eval instant at 50m changes(http_requests[50m])
 	{path="/biz"} 1
 
 eval instant at 50m changes(nonexistent_metric[50m])
+
+
+clear
+
+# Tests for increase().
```

Comment:
Tests -> Testdata (otherwise now you've got two comments saying "Tests for increase()".) Or maybe even keep this one and remove the second one, since it's only one function using this testdata at the moment.